src/View/Helper/NumberHelper.php

```diff
@@ -222,6 +222,7 @@ public function formatDelta($value, array $options = []): string
      * if $currency argument is not provided. If boolean false is passed, it will clear the
      * currently stored value. Null reads the current default.
      * @return string|null Currency
+     * @deprecated 4.1.0 Use setDefaultCurrency()/getDefaultCurrency() instead.
```

Comment:
```suggestion
     * @deprecated 3.9.0 Use setDefaultCurrency()/getDefaultCurrency() instead.
```

Reply:
if you check these methods exist in `4.0.0` too

Reply:
Yes but we just started to declare, no?

Reply:
no , since 4.0.0
see https://github.com/cakephp/cakephp/blob/4.0.0/src/I18n/Number.php#L275-L278